packages/data/addon/adapters/facts/elide.ts

```diff
@@ -60,16 +63,24 @@ export default class ElideFactsAdapter extends EmberObject implements NaviFactAd
     const columnsStr = columns.map(col => getElideField(col.field, col.parameters)).join(' ');
 
     const filterStrings = filters.map(filter => {
-      const { field, parameters, operator, values } = filter;
+      const { field, parameters, operator, values, type } = filter;
       const fieldStr = getElideField(field, parameters);
+      const operatorStr = OPERATOR_MAP[operator as FilterOperator] || `=${operator}=`;
+      let filterVals = values;
+      if (type === 'timeDimension' && filterVals.length === 2) {
+        const { start, end } = Interval.parseFromStrings(String(filterVals[0]), String(filterVals[1])).asMoments();
+        filterVals = [
+          start.format(ELIDE_API_DATE_FORMAT),
+          end?.format(ELIDE_API_DATE_FORMAT) || moment().format(ELIDE_API_DATE_FORMAT)
```

Comment:
Can end be undefined?

Reply:
`Interval.parseFromStrings` potentially returns an `undefined` end, but we don't expect it to be undefined unless the end macro is `next` I believe. Since we're using `[inclusive,inclusive]` intervals with Elide, that shouldn't be an issue. I've just asserted that end is defined to get around the type check.